### PR TITLE
feat(git-tag): add the ability to use a tag prefix

### PR DESCRIPTION
**Pull request by motin**
This is useful for monorepos that includes the source code to several npm packages, all of which are tagged in Git using a separate prefix.

### CODE REVIEW

**Comment by motin** [comment]
CI seems to be failing in general. The tests associated with this change are passing:
```
      ✓ Should expect git tag to match version (284ms)
      ✓ Should expect prefixed git tag to match version (286ms)
      ✓ Should expect git tag to exist (273ms)
      ✓ Should pass validation (793ms)
      ✓ Should pass validation when prefixed (794ms)
      ✓ Should not validate if tag-validation is disabled (794ms)
```

**Comment by hdorgeval** [comment]
Hi @motin, thank you very much for your contribution 👍  ; give me some time to finalize your PR and check for failing tests

**Comment by hdorgeval** [comment]
Hi @motin, I am ready on my side to merge your PR. I would like you to have a look at the last commit that document your feature: do you think it's enough documentation ?

File: README.md [hunk 1]
```diff
@@ -279,6 +279,16 @@ When all validations pass, publish-please will show you the exact content of the
         }
          ```
 
+    - if the git tag contains a prefix to the version like for example `foo-v0.0.42`, the only way for the git tag validation to work is to setup this prefix in the `.publishrc` file:
```
Comment (motin) [inline README.md]: Shorter version:
- if the git tag contains a prefix to the version like for example `foo-v0.0.42`, supply the prefix in the `.publishrc` file:

**Comment by motin** [comment]
@hdorgeval Amazin job, thank you for doing this!

File: README.md (hunk 1 above)
Comment (hdorgeval) [reply README.md]: thnks: 👍

Comment (motin) [reply README.md]: @hdorgeval This is just a suggestion. Feel free to ignore it, or push an update as you please :)

**Comment by motin** [comment]
> I would like you to have a look at the last commit that document your feature: do you think it's enough documentation ?

Docs look great! Yes I believe it is enough.

**Comment by hdorgeval** [comment]
Hi @motin , I just merged your PR, il will ping ou once I have pushed it to npm

**Comment by hdorgeval** [comment]
@motin, I have pushed v5.5.0 to npm.

**Comment by motin** [comment]
> @motin, I have pushed v5.5.0 to npm.

Thank you once again for your swift efforts! The new version is now timely included in https://github.com/mozilla/OpenWPM/pull/373 :)

**Pull request merged**